internal/controller/pkg/revision/runtime_function.go

```diff
@@ -199,6 +201,14 @@ func functionServiceOverrides() []ServiceOverride {
 		// FunctionComposer) can load balance across the endpoints.
 		// https://kubernetes.io/docs/concepts/services-networking/service/#headless-services
 		ServiceWithClusterIP(corev1.ClusterIPNone),
+		ServiceWithAdditionalPorts([]corev1.ServicePort{
+			{
+				Name:       grpcPortName,
+				Protocol:   corev1.ProtocolTCP,
+				Port:       grpcPort,
+				TargetPort: intstr.FromString(grpcPortName),
```

Comment:
we're setting the targetPort to the name of the gRPC port for a function's service in case the user has changed the pod/containers gRPC port via the deployment template, right? by using the name, we don't also have to plumb in here the specific value the user specified?